src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -262,7 +262,7 @@ public String update(Integer id, String json, boolean autoCommit) throws SQLExce
      *
      * @return number of stop times updated
      */
-    public int normalizeStopTimesForPattern(int id, int beginWithSequence) throws SQLException {
+    public int normalizeStopTimesForPattern(int id, int beginWithSequence, boolean interpolateStopTimes) throws SQLException {
```

Comment:
All other places have been updated, but I agree it seems smart to avoid breaking the existing API. I have added an override method with the old signature in https://github.com/conveyal/gtfs-lib/pull/399/commits/36f3043f11e761ea3a738f61174ec4ab5e363c31